test/test_SW_VegProd.cc

```diff
@@ -159,4 +159,604 @@ namespace {
+
+        /* Expect identical output to rSOILWAT2 (e.g., v5.3.1)
+         * NOTE: Command uses deprecated estimate_PotNatVeg_composition (rSOILWAT >= v.6.0.0)
+         ```{r}
+           rSOILWAT2::estimate_PotNatVeg_composition_old(
+             MAP_mm =  10 * clim1[["MAP_cm"]], MAT_C = clim1[["MAT_C"]],
+             mean_monthly_ppt_mm = 10 * clim1[["meanMonthlyPPTcm"]],
+             mean_monthly_Temp_C = clim1[["meanMonthlyTempC"]]
+           )
+         ```
+         */
+
+        // Loop through RelAbundanceL0 and test results
+        for(index = 0; index < 8; index++) {
+            EXPECT_NEAR(RelAbundanceL0[index], RelAbundanceL0Expected[index], tol6);
+        }
+
+        // Loop through RelAbundanceL1 and test results
+        for(index = 0; index < 5; index++) {
+            EXPECT_NEAR(RelAbundanceL1[index], RelAbundanceL1Expected[index], tol6);
+        }
+        
+        EXPECT_DOUBLE_EQ(grassOutput[0], 1.);
+        EXPECT_DOUBLE_EQ(grassOutput[1], 0.);
+        EXPECT_DOUBLE_EQ(grassOutput[2], 0.);
+
+        /*  ===============================================================
+                     Test with half of input values not "SW_MISSING"
+            ===============================================================  */
+        inputValues[succIndex] = .376;
+        inputValues[forbIndex] = SW_MISSING;
+        inputValues[C3Index] = .096;
+        inputValues[C4Index] = SW_MISSING;
+        inputValues[grassAnn] = SW_MISSING;
+        inputValues[shrubIndex] = .1098;
+        inputValues[treeIndex] = .0372;
+        inputValues[bareGround] = SW_MISSING;
+
+        /* Expect identical output to rSOILWAT2 (e.g., v5.3.1)
+         * NOTE: Command uses deprecated estimate_PotNatVeg_composition (rSOILWAT >= v.6.0.0)
+         ```{r}
+           rSOILWAT2::estimate_PotNatVeg_composition_old(
+             MAP_mm =  10 * clim1[["MAP_cm"]], MAT_C = clim1[["MAT_C"]],
+             mean_monthly_ppt_mm = 10 * clim1[["meanMonthlyPPTcm"]],
+             mean_monthly_Temp_C = clim1[["meanMonthlyTempC"]],
+             Succulents_Fraction = .376, fix_succulents = TRUE,
+             C3_Fraction = .096, fix_C3grasses = TRUE,
+             Shrubs_Fraction = .1098, fix_shrubs = TRUE,
+             Trees_Fraction = .0372, fix_trees = TRUE
+           )
+         ```
+         */
+
+        RelAbundanceL0Expected[succIndex] = 0.3760;
+        RelAbundanceL0Expected[forbIndex] = 0.3810;
+        RelAbundanceL0Expected[C3Index] = 0.0960;
+        RelAbundanceL0Expected[C4Index] = 0.0000;
+        RelAbundanceL0Expected[grassAnn] = 0.0000;
+        RelAbundanceL0Expected[shrubIndex] = 0.1098;
+        RelAbundanceL0Expected[treeIndex] = 0.0372;
+        RelAbundanceL0Expected[bareGround] = 0.0000;
+
+        RelAbundanceL1Expected[treeIndexL1] = 0.0372;
+        RelAbundanceL1Expected[forbIndexL1] = 0.1098;
+        RelAbundanceL1Expected[shrubIndexL1] = 0.7570;

```

Comment:
`RelAbundanceL1Expected[shrubIndexL1] = 0.7570;` is incorrect; this should be `0.1098` -- see line 302 with `RelAbundanceL0Expected[shrubIndex] = 0.1098;` and `inputValues[shrubIndex] = .1098;`